src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,125 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String SHORT_PASSWORD = "12345";
+    private static final String MIN_PASSWORD = "123456";
+    private static final String DEFAULT_USER_LOGIN = "Marcus";
+    private static final String DEFAULT_PASSWORD = "Marcus4ever";
+    private static final int DEFAULT_AGE = 20;
+    private static final int MIN_AGE = 18;
+    private static RegistrationServiceImpl registrationService;
+    private User testUser1;
+    private User testUser2;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        testUser1 = new User(DEFAULT_USER_LOGIN, DEFAULT_PASSWORD, DEFAULT_AGE);
+        testUser2 = new User(DEFAULT_USER_LOGIN, MIN_PASSWORD,DEFAULT_AGE);
+    }
+
+    @Test
+    void register_NullUser_NotOk() {
+        User nullUser = null;
+        assertThrows(RuntimeException.class, () -> {
+            registrationService.register(nullUser);
+        });
+    }
+
+    @Test
+    void register_setNullLogin_NotOk() {
```

Comment:
```suggestion
    void register_nullLogin_NotOk() {
```
without "set" because second part of the name it is statement. Fix everywhere